SECURITY.md

```diff
@@ -52,3 +52,81 @@ the code.
 - HD key derivation, local and Ledger, and all key-management functionality
 - Side-channel attack vectors with our implementations
   - e.g. key exfiltration based on time or memory-access patterns when decrypting privkey
+  
+## Disclosure Process
+
+The Cosmos SDK team uses the following disclosure process:
+
+1. Once a security report is received, the Cosmos SDK team works to verify the issue and confirm its severity level using CVSS.
+1. The Cosmos SDK team collaborates with the Tendermint and Gaia teams to determine the vulnerability’s potential impact on the Cosmos Hub and partners.
+1. Patches are prepared for eligible releases of Cosmos SDK in private repositories. See “Supported Releases” below for more information on which releases are considered eligible.
```

Comment:
```suggestion
1. Patches are prepared in private repositories for eligible releases of Cosmos SDK. See [Stable Releases](https://github.com/cosmos/cosmos-sdk/blob/master/STABLE_RELEASES.md) for a list of eligible releases.
```

Reply:
I think we are referencing the https://github.com/cosmos/cosmos-sdk/blob/master/STABLE_RELEASES.md file? (check my link strategy, do we use the full URL?)